akka-docs/rst/java/persistence.rst

```diff
@@ -119,7 +119,7 @@ about successful state changes by publishing events.
 When persisting events with ``persist`` it is guaranteed that the persistent actor will not receive further commands between
 the ``persist`` call and the execution(s) of the associated event handler. This also holds for multiple ``persist``
 calls in context of a single command. Incoming messages are :ref:`stashed <stash-java>` until the ``persist``
-is completed. You should be careful to not send more messages to a persistent actor than it can keep up with,
+is completed. You should be careful not to send more messages to a persistent actor than it can keep up with,
```

Comment:
This change is wrong: “not to” is only correct at the end of a sentence (i.e. with the infinitive form omitted), it is incorrect where the infinitive form still follows. Ask yourself what is inverted: the action of sending messages or being careful?